unstructured/ingest/v2/pipeline/interfaces.py

```diff
@@ -61,7 +61,7 @@ def _wrap_mp(self, input_kwargs: dict) -> Any:
 
     def _set_log_level(self, log_level: int):
         # Set the log level for each spawned process when using multiprocessing pool
-        logger.setLevel(log_level)
+        make_default_logger(log_level)
```

Comment:
This causes logs to be created for every process I think. Though I think it is worth it if it may surface errors better.

Reply:
Fixed this in the latest commit